log2ram.conf

```diff
@@ -1,49 +1,66 @@
-# The variable may be uncommented out, setting it to false, if "cp" is preferred over "rsync".
-# Currently, this option needs to be unset or set to true for "rsync" to run,
-# with "cp" available for fallback when "rsync" is missing.
-# In all other cases, setting USE_RSYNC to anything other than true will default to "cp".
+# Select the log syncing method between disk and RAM:
+# - 'rsync' is the default unless 'USE_RSYNC' is set to 'false'.
+# - If 'rsync' is unavailable, 'cp' is automatically used as a fallback.
+# - Set 'USE_RSYNC' to 'false' to explicitly use 'cp'.
 #USE_RSYNC=false
 
-# If there are some errors with available RAM space, a system mail will be send by default (see next parameter)
-# Change it to false and you will have only a log if there is no place on RAM anymore.
+# By default, if there is insufficient RAM space, a system notification email is sent.
+# Set this to 'false' to disable email notifications. (The error will still be logged locally).
 #NOTIFICATION=true
 
-# This is a one-line command for the error notification. 
-# By default, it's a email with the command `mail`, the body is passed in stdin.
-# So you can set any command(like shoutrrr) or custom script for the error notification.
+# Specify the command for sending error notifications.
+# By default, it uses the `mail` command to send an email, with the message body provided via stdin.
+# You can replace this with any command (such as 'shoutrrr') or a custom script for handling notifications.
 #NOTIFICATION_COMMAND=mail -s "Log2Ram Error on $HOSTNAME"
 
-# Variable for folders to put in RAM. You need to specify the real folder `/path/folder` , the `/path/hdd.folder` will
-# be automatically created. Multiple path can be separeted by `;`. Do not add the final `/` !
-# example : PATH_DISK="/var/log;/home/test/FolderInRam"
+# Specify the directories to be stored in RAM. List each directory using its absolute path, e.g., `/path/folder`.
+# Corresponding directories on the HDD, named `/path/hdd.folder`, will be automatically created for each listed path.
+# Separate multiple paths with a semicolon `;` and do not include a trailing slash at the end of the paths.
+# Example: PATH_DISK="/var/log;/home/test/FolderInRam"
 PATH_DISK="/var/log"
 
 # Should log2ram consider journald and do a log rotate before copying the log files back? Please note that for this
 # rsync is mandatory. Also make sure that you have configured SystemMaxUse in journald.conf, because the size of this
 # ram folder is will probably not be as large as journald will use by default
 JOURNALD_AWARE=true
 
-# **************** Zram backing conf  *************************************************
+# Set to 'true' to enable log rotation for journald logs before syncing. 
+# Note: 'rsync' must be used for this feature. 
+# Ensure 'SystemMaxUse' is configured in 'journald.conf' 
+#  - (To limit journald’s disk usage to be smaller than the log2ram RAM allocation).
+JOURNALD_AWARE=true
```

Comment:
Ins;t this block duplicated with line 35?

Reply:
Yes, thankyou for catching this! It has now been fixed in: [this commit](https://github.com/azlux/log2ram/pull/227/commits/feccb1d3c0b5e2b8727db988bb1c24f16b4cc8af)